src/pages/Mingi/Detail/Detail.scss

```
@@ -0,0 +1,258 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+.Detail {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+
+	header {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		width: 90%;
+		background-color: #f6f5f0;
+		font-family: "Montserrat Subrayada", sans-serif;
+	}
+
+	header h1 {
+		padding: 20px;
+		font-size: 20px;
+	}
+
+	nav ul {
+		display: flex;
+	}
+
+	nav ul li {
+		padding: 0 20px;
+		font-size: 15px;
+		list-style: none;
+		color: #9c9c98;
+	}
```

```suggestion
	nav ul {
		display: flex;
		
		  li {
		  padding: 0 20px;
		  font-size: 15px;
		  list-style: none;
		  color: #9c9c98;
	  }
	}
```

nav ul 안에 li를 네스팅해도 될 것 같습니다 ㅎㅎ